CHANGELOG.md

```diff
@@ -1,8 +1,22 @@
 # did-io ChangeLog
 
+## 0.8.0
+(Migrating from v0.7.0 will require renaming the relevant properties in existing
+DID Docs.)
```

Comment:
A parenthetical requirement looks odd.  In other packages I've make a top `### Notes` section with special important `BREAKING` issues.  This one could fit that pattern.

Reply:
@davidlehn np, I moved that note to one of the Changed items.